.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "modules/tests-sos"]
+	path = modules/tests-sos
+	url = git@github.com:openshmem-org/tests-sos.git
+	branch = main
```

Comment:
Should this point to main or a tagged commit? Should be fine, so long as we are ok with any changes made to main being pulled by users. Is there a possibility while working on changes for OpenSHMEM 1.x.y we would add tests to the main branch on tests-sos that we wouldn't necessarily want users to pull when building the test-sos repo with an older OpenSHMEM version. Please let me know if I didn't explain that well 😅

Reply:
Great question.  I prefer `main` align with the `main` of SOS, because the addition of a new test (for example) should continue to be tested throughout development cycle towards the next release.  However, when we do release we will likely want to point to a tag or branch here, so this should be added to our release checklist.

I think we should document how to handle things like this in the SOS wiki before merging this PR.  As another example, developers need to be aware that this change will likely cause some pull requests to have 2 parts: one for SOS and one for tests-sos.  This could be trickier than it sounds, because the SOS part will need to test against the tests-sos part, likely before the tests-sos part is merged...